src/native/minipal/utils.h

```diff
@@ -50,13 +50,19 @@
 #    define DISABLE_ASAN
 #endif
 
+#if defined(_MSC_VER)
```

Comment:
Nit: Could you please move this to be right after `define FALLTHROUGH` so that it is not in the middle of ASAN related stuff?